### PR TITLE
docs: add Chinese translation for the new jwe-decrypt plugin doc

### LINKED ISSUE #10801: docs: add Chinese translation for the new `jwe-decrypt` plugin doc
### Current State

Feature added in this PR: https://github.com/apache/apisix/pull/10252/files

Current docs: https://apisix.apache.org/zh/docs/apisix/next/plugins/jwe-decrypt/

### Desired State

Should have Chinese translation as well.

### PR DESCRIPTION
**Pull request by smileby**
### Description

add Chinese translation for the new jwe-decrypt plugin doc

Fixes #10801 

### Checklist

- [X] I have explained the need for this PR and the problem it solves
- [X] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [X] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Review by kayx23 — APPROVED** [review]
Thanks again

File: docs/zh/latest/plugins/jwe-decrypt.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## 描述
+
+`jwe-decrypt` 插件，用于解密 APISIX [Service](../terminology/service.md) 或者 [Route](../terminology/route.md) 请求中的 [JWE](https://datatracker.ietf.org/doc/html/rfc7516) 授权请求头。
+
+插件增加了一个 `/apisix/plugin/jwe/encrypt` 的 内部 API，提供给 JWE 加密使用。解密时，秘钥应该配置在 [Consumer](../terminology/consumer.md)。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/jwe-decrypt.md:35]:
```suggestion
插件增加了一个 `/apisix/plugin/jwe/encrypt` 的内部 API，提供给 JWE 加密使用。解密时，秘钥应该配置在 [Consumer](../terminology/consumer.md) 内。
```

File: docs/zh/latest/plugins/jwe-decrypt.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+## 属性
+
+Consumer 配置：
+
+| 名称          | 类型      | 必选项   | 默认值   | 有效值 | 描述                                                                    |
+|---------------|---------|-------|-------|-----|-----------------------------------------------------------------------|
+| key           | string  | True  |       |     | Consumer 的唯一 key                                                      |
+| secret        | string  | True  |       |     | 解密密钥。秘钥可以使用 [Secret](../terminology/secret.md) 资源保存在密钥管理服务中。（最小 32 位） |
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/jwe-decrypt.md:44]:
```suggestion
| secret        | string  | True  |       |     | 解密密钥。秘钥可以使用 [Secret](../terminology/secret.md) 资源保存在密钥管理服务中（最小 32 位）|
```

File: docs/zh/latest/plugins/jwe-decrypt.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+| is_base64_encoded | boolean | False | false |     | 如果密钥是 base64 编码，则需要配置为 true                                           |
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/jwe-decrypt.md:45]:
```suggestion
| is_base64_encoded | boolean | False | false |     | 如果密钥是 Base64 编码，则需要配置为 `true`                                      |
```

File: docs/zh/latest/plugins/jwe-decrypt.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+
+Route 配置：
+
+| 名称             | 类型      | 必选项   | 默认值           | 描述                                                                         |
+|----------------|---------|-------|---------------|----------------------------------------------------------------------------|
+| header         | string  | False | authorization | 指定请求头，用于获取加密令牌                                                             |
+| forward_header | string  | False | authorization | 传递给 Upstream 的请求头名称                                                        |
+| strict         | boolean | False | true          | 如果为配置为 true，请求中缺失 JWE token 则抛出 403 异常。如果为 false, 在缺失 JWE token 的情况下不会抛出异常 |
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/jwe-decrypt.md:53]:
```suggestion
| strict         | boolean | False | true          | 如果为配置为 `true`，请求中缺失 JWE token 则抛出 `403` 异常。如果为 `false`, 在缺失 JWE token 的情况下不会抛出异常 |
```

File: docs/zh/latest/plugins/jwe-decrypt.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+curl http://127.0.0.1:9180/apisix/admin/routes/jwenew -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+    "uri": "/apisix/plugin/jwe/encrypt",
+    "plugins": {
+        "public-api": {}
+    }
+}'
+```
+
+向 API 发送一个请求，将 Consumer 中配置的密钥，以参数的方式传递给 URI，用于加密 payload 中的一些数据
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/jwe-decrypt.md:105]:
```suggestion
向 API 发送一个请求，将 Consumer 中配置的密钥，以参数的方式传递给 URI，用于加密 payload 中的一些数据。
```

File: docs/zh/latest/plugins/jwe-decrypt.md [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+
+```shell
+curl -G --data-urlencode 'payload={"uid":10000,"uname":"test"}' 'http://127.0.0.1:9080/apisix/plugin/jwe/encrypt?key=user-key' -i
+```
+
+您应该看到类似于如下内容的响应结果，其中 JWE 加密的数据位于响应体中
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/jwe-decrypt.md:111]:
```suggestion
您应该看到类似于如下内容的响应结果，其中 JWE 加密的数据位于响应体中：
```

File: docs/zh/latest/plugins/jwe-decrypt.md [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+  "json": null,
+  "method": "GET",
+  "origin": "127.0.0.1, 119.143.79.94",
+  "url": "http://127.0.0.1/anything/hello"
+}
+```
+
+## 删除插件
+
+要删除 `jwe-decrypt` 插件，你可以从插件配置中删除插件对应的 JSON 配置，APISIX 会自动加载，您不需要重新启动即可生效。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/jwe-decrypt.md:168]: keep "你" or "您" in the whole article, make the word consistently

**Review by kayx23 — APPROVED** [review]

**Comment by smileby** [comment]
@SylviaBABY Thank you.  done with the repair.

**Review by SylviaBABY — APPROVED** [review]

**Comment by yzeng25** [comment]
@smileby @kayx23 could you double check the error?

image

**Comment by kayx23** [comment]
It's related to docker and not related to the update in this PR, as the CI all passed in the last commit.

I'll keep an eye out to see if this error occurs repetitively and report if it does.

**Pull request merged**